pkg/sysregistriesv2/shortnames_test.go

```diff
@@ -0,0 +1,307 @@
+package sysregistriesv2
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/containers/image/v5/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseShortNameMode(t *testing.T) {
+	tests := []struct {
+		input    string
+		result   types.ShortNameMode
+		mustFail bool
+	}{
+		{"", types.ShortNameModePermissive, false}, // default to permissive
+		{"disabled", types.ShortNameModeDisabled, false},
+		{"enforcing", types.ShortNameModeEnforcing, false},
+		{"permissive", types.ShortNameModePermissive, false},
+		{"xxx", -1, true},
+	}
+
+	for _, test := range tests {
+		shortName, err := parseShortNameMode(test.input)
+		if test.mustFail {
+			require.Error(t, err)
```

Comment:
Absolutely non-blocking: This one can be `assert.Error` because nothing will break if it is not true, we can just continue with another case. (OTOH, the test fails anyway, so, not at all worth worrying about.)

(Several other instances I won’t bother to point out individually.)